sdk/ai/Azure.AI.VoiceLive/tsp-location.yaml

```diff
@@ -0,0 +1,5 @@
+directory: specification/ai/data-plane/VoiceLive
+commit: fbf83df305bfa7597dcc2f248e9f0e49efb5cd05
+repo: rhurey/azure-rest-api-specs
+additionalDirectories: 
```

Comment:
Please use the new emitter, e.g. https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/eventgrid/Azure.Messaging.EventGrid.SystemEvents/tsp-location.yaml#L4

Reply:
Ah, I'd thought that went in the tsp-config.yaml in the specs repro....